spec/waypoint_spec.js

```diff
@@ -1138,11 +1138,19 @@ describe('<Waypoint>', function() {
       this.parentStyle.height = 'auto';
       this.parentStyle.overflow = 'visible';
 
+      // This is only here to try and confuse the _findScrollableAncestor code.
+      document.body.style.overflow = 'auto';
+
       // Make the spacers large enough to make the Waypoint render off-screen
       this.topSpacerHeight = window.innerHeight + 1000;
       this.bottomSpacerHeight = 1000;
     });
 
+    afterEach(() => {
+      // Reset body style
+      document.body.style.overflow = '';
```

Comment:
Ah! This is the "right" way to do things. I initially tried `delete` and setting to `undefined` here before. 

http://stackoverflow.com/questions/3506050/how-to-reset-the-style-properties-to-their-css-defaults-in-javascript